filter_plugins/ec2_custom.py

```diff
@@ -1,5 +1,15 @@
 from ansible import errors
 
+# list all known app component names.
+VALID_APP_COMPONENT_NAMES = ["frontend",
```

Comment:
how do you propose injecting those component names to ansible? considering they are named differently across the different apps?
in jenkins?